Source/WebCore/page/Quirks.cpp

```diff
@@ -1501,7 +1502,6 @@ bool Quirks::needsVideoShouldMaintainAspectRatioQuirk() const
 {
     if (!needsQuirks())
         return false;
-
```

Comment:
Unnecessary change, please revert.

Reply:
@stwrt Whitespace changes should be omitted from the change.